terraform/modules/account_mapping/account_mapping.tf

```diff
@@ -0,0 +1,77 @@
+variable "google_account_name" {
```

Comment:
Please restore the comments explaining why we're associating all these objects to each other as well as the links to GCP documentation.